build.sbt

```diff
@@ -152,14 +156,8 @@ lazy val commonSettings = List(
     "-Dsbt.execute.extrachecks=true" // Avoid any deadlocks.
   ),
   scriptedBufferLog := false,
-  crossSbtVersions := List("0.13.17", "1.2.8"),
+  crossSbtVersions := List("1.2.8"),
   sbtVersion in pluginCrossBuild := "1.2.8",
```

Comment:
Should we use 1.3.0 everywhere, for simplicity?

Reply:
If we do that, then users of sbt-scalajs-bundler would be forced to upgrade to sbt 1.3.x. Is that something we want to do?

Even if yes, I believe that should be in a separate PR.

Reply:
No, ok for staying on 1.2